media-libs/libwebm/libwebm-1.0.0.27.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="The SDK of the VP8 video codec developed by Google"
```

Comment:
According to the [homepage][0], libwebm is a "WebM file parser" and has nothing to do with the VP8/9 codecs, that's the job of libvpx.

[0]: https://www.webmproject.org/code/

Reply:
Oops, sorry guys. I stand corrected :)

Reply:
Thanks, I'll fix this later :)